central/complianceoperator/v2/report/manager/complianceReportgenerator/type.go

```diff
@@ -7,11 +7,12 @@ import (
 )
 
 type ComplianceReportRequest struct {
-	ScanConfigID   string
-	Notifiers      []*storage.NotifierConfiguration
-	ClusterIDs     []string
-	Profiles       []string
-	ScanConfigName string
-	Ctx            context.Context
-	SnapshotID     string
+	ScanConfigID       string
+	Notifiers          []*storage.NotifierConfiguration
+	ClusterIDs         []string
+	Profiles           []string
+	ScanConfigName     string
+	Ctx                context.Context
+	SnapshotID         string
+	NotificationMethod storage.ComplianceOperatorReportStatus_NotificationMethod
```

Comment:
make this a pointer and handle the nil in the converters.  The reason is if the field is not populated in the database it will try to read a null into the field and the query will fail.  (I just recently discovered this)

See https://github.com/stackrox/stackrox/pull/13073 for some details.